nginx.master.conf

```diff
@@ -283,7 +283,7 @@ http {
             proxy_pass http://log/;
         }
 
-        location ~ ^/system/logs/v1/(slave|agent)/(?<agentid>[0-9a-zA-Z-]+)(?<url>.*)$ {
+        location ~ ^/system/v1/logs/v1/(slave|agent)/(?<agentid>[0-9a-zA-Z-]+)(?<url>.*)$ {
```

Comment:
yes, we had this discussion here. We will change `/system/health` to `/system/v1/health/v1`